twentytwentyone-blocks/style.css

```
@@ -103,7 +103,7 @@ body {
 
 	.wp-site-blocks .alignfull {
 		transform: translateX(0px);
-		width: 100% + var(--wp--custom--spacing--horizontal));
+		width: 100% + var(--wp--custom--spacing--horizontal);
```

What does this rule mean? The width of the block should be 100% of its container plus some horizontal spacing? 

Also I didn't know you could do a math operation outside of calc. The browser does appear to compute it and an alignfull image appears correctly in my testing. But setting this to `100%` appears to have the same effect.

```suggestion
		width: 100%;
```

---

yeah, I just saw that the extra parenthesis was breaking the alignfull images and that was a quick fix but I'm going to make a separate PR (https://github.com/WordPress/theme-experiments/issues/104) for the alignments since they are not up to date with TT1's. I just left this here as a quick fix in the meantime